clients/go/gengo.go

```diff
@@ -177,7 +176,7 @@ func (c *WagClient) SetCircuitBreakerSettings(settings CircuitBreakerSettings) {
 }
 
 // SetTimeout sets a timeout on all operations for the client. To make a single request
-// with a timeout use context.WithTimeout as described here: https://godoc.org/golang.org/x/net/context#WithTimeout.
+// with a shorter timeout use context.WithTimeout as described here: https://godoc.org/golang.org/x/net/context#WithTimeout.
```

Comment:
shorter than the server timeout? shorter than what?

Reply:
shorter than default client timeout

Reply:
sweet